lochness/cleaner/__init__.py

```diff
@@ -24,12 +40,12 @@ def get_ok2remove_df_from_s3_log(phoenix_root: Path,
     df_okay_to_remove = df_okay_to_remove[
             ~df_okay_to_remove.source.isin(df_not_okay_to_remove.source)]
 
+    # XNAT
     # since lochness.xnat pulls a zip file to a temporary file and extracts
-    # to a directory, a root directory of the saved MRI data needs to be added 
-    mri_root_names = df_okay_to_remove[
-            df_okay_to_remove['datatypes']=='mri'][
-                    'source'].str.extract(
-                            r'([\w/]*/mri/\w+)/').drop_duplicates()
+    # to a directory, a root directory of the saved MRI data needs to be added
+    mri_df = df_okay_to_remove[df_okay_to_remove['datatypes'] == 'mri']
+    mri_root_names = mri_df['source'].str.extract(r'([\w/]*/mri/\w+)/')
+    mri_root_names.drop_duplicates(inplace=True)
```

Comment:
Did you break it down to ease debugging?

Reply:
Yes, exactly.